### DISCUSSION TITLE
31 Mic - microphone sensitivity

### DISCUSSION THREAD

**Discussion by RoadRangerX** [egzumer/uv-k5-firmware-custom#352] [Q&A]
Does this work (been tested?) if using an external mic?
The packet interface I'm using is set for its lowest output (1%) but seems to still be a bit high.
The mic gain presently bottoms out at +1.1dB, can this be made to go lower?

**Comment by ErikS-web** [comment]
Tip: Use 2 resistors, to lower your signal, or a potentiometer
